automl/README.rst

```diff
@@ -104,3 +104,32 @@ Next Steps
    API to see other available methods on the client.
 -  Read the `Product documentation`_ to learn
    more about the product and see How-to Guides.
+
+Making & Testing Local Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to make changes to this library, here is how to set up your
+development environment:
+
+1. Make sure you have `virtualenv`_ installed and activated as shown above.
+2. Run the following one-time setup (it will be persisted in your virtualenv):
+
+.. code-block:: console
+
+    pip install -r ../docs/requirements.txt
+    pip install -U mock pytest
```

Comment:
This library uses [nox](https://nox.thea.codes/en/stable/) for testing. Nox creates a new virtualenv for each test run and installs the necessary dependencies).
```
pip install --user --upgrade nox
```

`nox` will run all of the nox sessions defined in the [noxfile.py](https://github.com/googleapis/google-cloud-python/blob/master/automl/noxfile.py). You can run a specific session by doing `nox -s <session_name>`. For example, to just run the Python 2.7 system tests you would do `nox -s system-2.7`.

Reply:
Will do. Should I include `pytest mock` as requirements as well just to be safe?

Reply:
Yes, feel free to keep that info as well.